tianshou/policy/imitation/td3_bc.py

```diff
@@ -1,49 +1,46 @@
-    :param float tau: param for soft update of the target network. Default to 0.005.
-    :param float gamma: discount factor, in [0, 1]. Default to 0.99.
-    :param float exploration_noise: the exploration noise, add to the action.
-        Default to ``GaussianNoise(sigma=0.1)``
-    :param float policy_noise: the noise used in updating policy network.
-        Default to 0.2.
-    :param int update_actor_freq: the update frequency of actor network.
-        Default to 2.
-    :param float noise_clip: the clipping range used in updating policy network.
-        Default to 0.5.
-    :param float alpha: the value of alpha, which controls the weight for TD3 learning
+    :param actor_optim: the optimizer for actor network.
+    :param critic: the first critic network. (s, a -> Q(s, a))
+    :param critic_optim: the optimizer for the first critic network.
+    :param action_space: Env's action space. Should be gym.spaces.Box.
+    :param critic2: the second critic network. (s, a -> Q(s, a)).
+        If None, use the same network as critic (via deepcopy).
+    :param critic2_optim: the optimizer for the second critic network.
+        If None, clone critic_optim to use for critic2.parameters().
+    :param tau: param for soft update of the target network.
+    :param gamma: discount factor, in [0, 1].
+    :param exploration_noise: add noise to action for exploration.
+        This is useful when solving "hard exploration" problems.
+        "default" is equivalent to GaussianNoise(sigma=0.1).
+    :param policy_noise: the noise used in updating policy network.
+    :param update_actor_freq: the update frequency of actor network.
+    :param noise_clip: the clipping range used in updating policy network.
+    :param alpha: the value of alpha, which controls the weight for TD3 learning
         relative to behavior cloning.
-    :param bool reward_normalization: normalize the reward to Normal(0, 1).
-        Default to False.
-    :param bool action_scaling: whether to map actions from range [-1, 1] to range
-        [action_spaces.low, action_spaces.high]. Default to True.
-    :param str action_bound_method: method to bound action to range [-1, 1], can be
-        either "clip" (for simply clipping the action) or empty string for no bounding.
-        Default to "clip".
-    :param Optional[gym.Space] action_space: env's action space, mandatory if you want
-        to use option "action_scaling" or "action_bound_method". Default to None.
-    :param lr_scheduler: a learning rate scheduler that adjusts the learning rate in
-        optimizer in each policy.update(). Default to None (no lr_scheduler).
+    :param action_scaling: whether to map actions from range [-1, 1]
+        to range[action_spaces.low, action_spaces.high].
+    :param action_bound_method: method to bound action to range [-1, 1],
+        can be either "clip" (for simply clipping the action)
+        or empty string for no bounding. Only used if the action_space is continuous.
```

Comment:
or `None` for no bounding

Reply:
missing obsevation_space?

Reply:
Made docstrings for these params the same as in DDPG